tests/components/sharkiq/test_shark_iq.py

```diff
@@ -58,122 +81,107 @@ def _set_property(self, property_name, value):
 
 async def _async_set_property(self, property_name, value):
     """Set a property locally without hitting the API."""
+    print(f"Setting {id(self)}/{property_name} to {value}")
     _set_property(self, property_name, value)
 
 
-def _get_mock_shark_vac(ayla_api: AylaApi) -> SharkIqVacuum:
-    """Create a crude sharkiq vacuum with mocked properties."""
-    shark = SharkIqVacuum(ayla_api, SHARK_DEVICE_DICT)
-    shark.properties_full = deepcopy(SHARK_PROPERTIES_DICT)
-    return shark
+async def async_nop(*args, **kwargs):
+    """Don't do nothin'."""
+
+
+@pytest.fixture(autouse=True)
+@patch("sharkiqpy.ayla_api.AylaApi", MockAyla)
+@patch.object(SharkIqUpdateCoordinator, "_async_update_vacuum", new=async_nop)
```

Comment:
Don't patch integration details. We want to patch in the periphery. Patch the library client or client method(s).

Reply:
The default new when using `patch` will be an `AsyncMock` if the target is a coroutine function.